src/Firewall/Helpers.php

```diff
@@ -783,7 +781,5 @@ function get_session_id(): string
  */
 function create_session_id(): string
 {
-    $hash =  rand() . 'ej;1zj47vu;3e;31g642941ek62au/41' . time();
-
-    return md5($hash);
-}
\ No newline at end of file
+    return md5(rand() . SESSION_ID_SALT . time());
```

Comment:
Wouldn't it be better to use a CSPRNG here (like `bin2hex(random_bytes(25))`)?

Reply:
Definitely **yes**.

This PR only fixes small and big problems.